.github/workflows/ci_suite.yml

```diff
@@ -41,6 +41,7 @@ jobs:
           bash tools/ci/check_filedirs.sh tgstation.dme
           bash tools/ci/check_changelogs.sh
           bash tools/ci/check_grep.sh
+          bash skyrat/tools/skyrat_check_grep.sh
```

Comment:
Is there a way to add comments in these files? If so, I'd appreciate a `// SKYRAT EDIT - Running the Skyrat Check grep` comment (not necessarily phrased like this, but you know what I mean), to save us some time later when there's inevitably conflicts in this file.